app/frontend/src/features/profile/edit/ProfileForm.tsx

```diff
@@ -143,31 +276,20 @@ export default function EditProfileForm() {
                 />
               )}
             />
-            <Controller
-              control={control}
-              defaultValue={user.aboutMe}
+            <ProfileTextInput
+              label={ABOUT_ME}
               name="aboutMe"
-              render={({ onChange, value }) => (
-                <ProfileMarkdownInput
-                  label={ABOUT_ME}
-                  onChange={onChange}
-                  value={value}
-                />
-              )}
+              defaultValue={user.aboutMe}
+              inputRef={register}
+              className={classes.field}
             />
-            <Controller
-              control={control}
-              defaultValue={user.aboutPlace}
+            <ProfileTextInput
+              label={ABOUT_HOME}
               name="aboutPlace"
-              render={({ onChange, value }) => (
-                <ProfileMarkdownInput
-                  label="About my place"
-                  onChange={onChange}
-                  value={value}
-                />
-              )}
+              defaultValue={user.aboutPlace}
+              inputRef={register}
+              className={classes.field}
             />
-
             <Controller
```

Comment:
I noticed that this (and other dropdowns) seem to have a different font size compared to the rest of the text fields. Do you mind fixing it while you're on the page please?